docs/commands/InModuleScope.mdx

```diff
@@ -73,6 +73,56 @@ the PowerShell session, because the module only exported
 Using InModuleScope allowed this call to
 "PrivateFunction" to work successfully.
 
+### EXAMPLE 2
+
+```powershell
+# The script module:
+
+function PublicFunction
+
+    # Does something
+
+}
+
+function PrivateFunction ($MyParam)
+
+    return $MyParam
+}
+
+Export-ModuleMember -Function PublicFunction
+
+# The test script:
+
+Describe 'Testing MyModule' {
+    BeforeAll {
+        Import-Module MyModule
+    }
+
+    It 'passing in parameter' {
+        $SomeVar = 123
+      
+            $MyVar | Should -Be 123
+        }
+    }
+
+    It 'accessing whole testcase in module scope' -TestCases @(
+        @{ Name = 'Foo'; Bool = $true }
+    ) {
+        # Passes the whole testcase-dictionary available in $_ to InModuleScope
+
+        InModuleScope 'MyModule' -Parameters $_ {
```

Comment:
Where did the empty line come from? Source:

```
            # Passes the whole testcase-dictionary available in $_ to InModuleScope
            InModuleScope 'MyModule' -Parameters $_ {
                $Name | Should -Be 'Foo'
```

Reply:
Fixed as well